package.json

```diff
@@ -21,10 +21,21 @@
     "file",
     "system"
   ],
-  "author": "jillix <contact@jillix.com>",
+  "author": {
+    "name": "jillix",
+    "email": "contact@jillix.com"
+  },
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/jillix/engine-form/issues"
   },
-  "homepage": "https://github.com/jillix/engine-form"
+  "homepage": "https://github.com/jillix/engine-form",
+  "gitHead": "c26295b9bb7b4d9134a763b118addf08dfd8563a",
```

Comment:
Can you please clean this up? You probably edited the npm module and then overriden everything in a clone of this repository.